### PR TITLE
Add description field to package metadata

### PR DESCRIPTION
**Pull request by earlbread**
This patch adds the description field to package metadata. (#100)

### CODE REVIEW

File: test/Nirum/Targets/PythonSpec.hs [hunk 1]
```diff
@@ -37,7 +37,11 @@ import Nirum.Constructs.TypeExpression ( TypeExpression ( ListModifier
                                        )
 import Nirum.Package (Package (metadata, modules), resolveBoundModule)
 import Nirum.Package.Metadata ( Author (Author, email, name, uri)
-                              , Metadata (Metadata, authors, target, version)
+                              , Metadata (Metadata
```
Comment (earlbread) [inline test/Nirum/Targets/PythonSpec.hs]: Ok, I'll update it. Thank you!

File: src/Nirum/Targets/Python.hs [hunk 2]
```diff
@@ -1207,6 +1213,10 @@ setup(
     pName = packageName $ target metadata'
     pVersion :: Code
     pVersion = SV.toText $ version metadata'
+    pDescription :: Code
+    pDescription = case description metadata' of
+                     Just value -> T.intercalate "" ["'", value, "'"]
```
Comment (earlbread) [inline src/Nirum/Targets/Python.hs]: You're right! How about this change?
```haskell
pDescription = case description metadata' of
                  Just value -> stringLiteral value
 ```

Comment (dahlia) [inline src/Nirum/Targets/Python.hs]: It's why `stringLiteral` function exists. Seems correct.

Comment (earlbread) [inline src/Nirum/Targets/Python.hs]: So I'll fix like above. Thank you!

**Comment by earlbread** [comment]
@dahlia I fixed you mentioned. Thank you!

**Pull request merged**